.vscode/settings.json

```diff
@@ -1,28 +1,25 @@
 {
-	"deno.enablePaths": ["scripts/deno"],
-	"deno.lint": true,
-	"deno.unstable": false,
-    "eslint.validate": ["typescript", "typescriptreact"],
-    "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
-    },
-    "eslint.workingDirectories": [
-        "./dotcom-rendering"
-	],
-    "files.watcherExclude": {
-        "**/target": true
-    },
-    "workbench.colorCustomizations": {
-        "titleBar.activeBackground": "#648FFF",
-        "titleBar.activeForeground": "#000000"
-	},
-	"jest.jestCommandLine": "yarn workspaces run test",
-    "typescript.tsdk": "node_modules/typescript/lib",
-    "gitlens.advanced.blame.customArguments": [
-        "--ignore-revs-file",
-        ".git-blame-ignore-revs"
-    ],
-    "git.branchProtection": [
-        "main"
-    ]
+  "deno.enablePaths": ["scripts/deno"],
+  "deno.lint": true,
+  "deno.unstable": false,
+  "eslint.validate": ["typescript", "typescriptreact"],
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "eslint.workingDirectories": ["./dotcom-rendering"],
+  "files.watcherExclude": {
+    "**/target": true
+  },
+  "workbench.colorCustomizations": {
+    "titleBar.activeBackground": "#648FFF",
+    "titleBar.activeForeground": "#000000"
+  },
+  "jest.jestCommandLine": "yarn workspaces run test",
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "gitlens.advanced.blame.customArguments": [
+    "--ignore-revs-file",
+    ".git-blame-ignore-revs"
+  ],
+  "git.branchProtection": ["main"],
+  "cSpell.words": ["Lightbox"]
```

Comment:
We recommend a spell checker, so I had to add this as it's not recognised as a word